static/CLAUDE.md

```diff
@@ -1 +0,0 @@
-AGENTS.md
```

Comment:
for some reason this diff is rendering weirdly, but the file contains `@AGENTS.md`